.github/workflows/deploy.yml

```diff
@@ -0,0 +1,48 @@
+name: Deploy
+
+on:
+  workflow_run:
+    workflows: [CI]
+    types: [completed]
+    branches: [main]
+
+jobs:
+  deploy:
```

Comment:
You may need to add a [`on-success:` property](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-a-workflow-based-on-the-conclusion-of-another-workflow).

If someone manages to merge broken code into `main` (not that they should be able to), we could potentially generate a bad output and deploy something completely broken to the user-facing Developer Docs. So it's better to check that the tests pass 🤞

Reply:
@theseanything this convo has been resolved but I don't see any changes - am I missing something?

Reply:
Thanks, makes sense